sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/MarshalSpec.kt

```diff
@@ -18,19 +18,12 @@ package com.squareup.sqldelight
 import com.squareup.javapoet.ClassName
 import com.squareup.javapoet.FieldSpec
 import com.squareup.javapoet.MethodSpec
+import com.squareup.javapoet.MethodSpec.Builder
 import com.squareup.javapoet.ParameterSpec
 import com.squareup.javapoet.TypeName
 import com.squareup.javapoet.TypeSpec
-import com.squareup.sqldelight.model.Table
-import com.squareup.sqldelight.model.adapterField
-import com.squareup.sqldelight.model.adapterType
-import com.squareup.sqldelight.model.constantName
-import com.squareup.sqldelight.model.isHandledType
-import com.squareup.sqldelight.model.isNullable
-import com.squareup.sqldelight.model.javaType
-import com.squareup.sqldelight.model.marshaledValue
-import com.squareup.sqldelight.model.methodName
-import com.squareup.sqldelight.model.paramName
+import com.squareup.sqldelight.model.*
```

Comment:
there's a setting to turn off `*` imports under I think `Preferences -> Code Style -> Kotlin -> Imports`